manifests/params.pp

```diff
@@ -486,6 +486,6 @@
 
   ## Additional graceful failures
   if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '4' and $facts['os']['name'] != 'Amazon' {
```

Comment:
This should be updated to check for RedHat less than 7

Reply:
Missed this one. Thanks for pointing out @david22swan